content/doc/developer/publishing/requesting-hosting.adoc

```diff
@@ -62,12 +63,42 @@ Alternately, you can configure your plugin to be released automatically whenever
 In this case you only need write permission to GitHub; you never need local credentials.
 Follow link:/redirect/continuous-delivery-of-plugins[this guide] for details.
 
-== Categorize the Plugin
+=== Automate your GitHub repository
+
+==== Activate GitHub actions
+
+During the hosting process, you have included a shared workflow to generate changelogs automatically. Publishing changelogs as GitHub release notes is a good practice to keep your users informed about the changes in your plugin.
+
+To activate the workflow, you need to go to the `Actions` tab of your GitHub repository and click on the `I understand my workflows, go ahead and enable them` button.
+Follow the next section to create the necessary labels for your pull requests.
+
+==== Create the default labels
+
+Use the following script to create the default labels for your GitHub repository, used to categorize issues and pull requests and generate changelogs automatically:
+
+The script makes use of `gh`, you may need to install and link:[authenticate](https://cli.github.com/manual/gh_auth_login) first, if you haven't already.
```

Comment:
> The script makes use […], you may need to install and authenticate […]

These are two independent clauses, each with a subject and verb, without a coordinating conjunction; i.e., a [run-on sentence](https://www.niu.edu/writingtutorial/punctuation/run-on-sentences.shtml). Suggest using a semicolon instead of a comma or adding a coordinating conjunction.